net/src/parse.rs

```diff
@@ -79,6 +79,43 @@ impl ParseHeader for Reader<'_> {
     }
 }
 
+// Trait ParseHeader above requires its second generic parameter to implement From<T>, leading in
```

Comment:
Why is this in parse.rs?  Shouldn't it be in mod.rs or maybe its own file?  No need to change it now, but in a future PR we can move it.